reqs/requirements-dev-contrib.txt

```diff
@@ -4,3 +4,4 @@ invoke>=2.0.0
 mypy==1.3.0
 pre-commit>=2.21.0
 ruff==0.0.271
+pyarrow
```

Comment:
`pyarrow` is required for `pandas` Parquet tests, which are non-backend specific.

Reply:
`FAILED tests/integration/test_script_runner.py::test_docs[how_to_connect_to_in_memory_data_using_pandas]` This is the result

Reply:
https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/fluent/in_memory/how_to_connect_to_in_memory_data_using_pandas

Reply:
Consider also moving to requirements-dev-lite.txt or a new requirements-dev-parquet.txt